### ISSUE TITLE
CLI ignores --no-source-map flag in many-to-many mode

### ISSUE THREAD

**Issue by ansem78** [sass/dart-sass#1216]
I'm using the CLI in many-to-many mode with --no-source-map flag. However, source maps code is still embedded at the end of each generated CSS file. In my package.json:

```
"scripts": {
    "compile": "sass --style=expanded --no-source-map --no-error-css --stop-on-error scss/:dist/"
}
```
 
In one-to-one mode, --no-source-map flag works.

**Comment by jathak** [comment]
What build system / Dart Sass version are you using?

I'm not able to replicate this with the CLI for the latest version (1.32.5).

**Comment by ansem78** [comment]
Sorry. I investigated more deep and I found that it was a postcss "bug" (or "feature", I don't know), not a sass one. I use sass to build in many-to-many mode without source maps. Then, I use postcss with autoprefixer plugin. From postcss docs, i can use --map to generate source maps, or --no-map if I don't want source maps. My script did not pass either a --map or a --no-map flag. So, I expected no source map will be generated. It run sequentially sass and postcss autoprefixer. No matter what flag I pass sass for source maps: postcss always add inline source map to the generated CSS file without explicit --no-map flag. Because I also use clean-css for minification, I have disabled source map for unminified CSS and I generate it only for minified one. It seems it was a sass bug because I have 3 SASS files to compile, each one with its own script. But I also have a script that run sass for all the 3 files, and another that run the full build (sass, autoprefixer, rtl, minification). Only this last one uses many-to-many mode and autoprefixer, that causes the problem.
